src/Field/SurveyLikertField.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace GFExcel\Field;
+
+/**
+ * A field transformer for the `likert` field from the Gravity Forms Survey Plugin.
+ * @since $ver$
+ * @see \GF_Field_Likert
+ */
+class SurveyLikertField extends SeparableField {
+	/**
+	 * @inheritdoc
+	 * @var \GF_Field_Likert
+	 */
+	protected $field;
+
+	/**
+	 * @inheritdoc
+	 * @since $ver$
+	 */
+	public function getFieldValue( $entry, $input_id = '' ) {
+		if ( ! $this->useScoreOutput() ) {
+			return parent::getFieldValue( $entry, $input_id );
+		}
+
+		$value = $entry[ $input_id ?: $this->field->id ] ?? ':';
+		// If this is not a multi row value; add a colon, so we can still get $column.
+		if ( strpos( $value, ':' ) === false ) {
+			$value = ':' . $value;
+		}
+
+		[ , $column ] = explode( ':', $value );
+
+		foreach ( $this->field->choices as $choice ) {
+			if ( ( $choice['value'] ?? '' ) === $column ) {
+				return $choice['score'];
+			}
+		}
+
+		return 0; // choice not found, so value is 0
```

Comment:
This return value isn't defined as a possible return type in `AbstractField::getFieldValue()`.